.graphqlconfig

```diff
@@ -1,5 +1,5 @@
 {
-  "schemaPath": "libraries/graphql_schemas/schema.graphql",
+  "schemaPath": "schema.graphql",
```

Comment:
Why do you change this? I was thinking it can get bigger or more files so that's why I created a folder inside of libraries.